apis/sgv2-restapi/src/main/java/io/stargate/sgv2/restapi/service/resources/Sgv2RowsResourceApi.java

```diff
@@ -96,7 +96,10 @@ Uni<RestResponse<Object>> getRowWithWhere(
       @Parameter(name = "raw", ref = RestOpenApiConstants.Parameters.RAW) @QueryParam("raw")
           final boolean raw,
       @Parameter(name = "sort", ref = RestOpenApiConstants.Parameters.SORT) @QueryParam("sort")
-          final String sort);
+          final String sort,
+      @Parameter(name = "optimize_map", ref = RestOpenApiConstants.Parameters.OPTIMIZE_MAP)
+          @QueryParam("optimize_map")
+          final boolean optimizeMap);
```

Comment:
This should be `Boolean` since otherwise there will always be a value and defaults are never used?
If using wrapper, `null` value (missing query parameter) would use the default.

Reply:
Good catch. Fixed this now in all the API entry points and also started adding integration tests now, which also caught this issue. I'll add more integration tests and ll update.

For the integration tests, I'm following the pattern that you used for CQL enabled/disabled flag tests. Hope that's fine. Please review the Integration tests model I'm setting for this as well when you get a chance.